.github/workflows/deliver-ubuntu.yml

```diff
@@ -0,0 +1,110 @@
+name: delivery / ubuntu
+
+on:
+  release:
+    types:
+      - released
```

Comment:
Could we add mechanism for triggering it with a workflow-dispatch, similar to what we have here: https://github.com/buildpacks/pack/blob/1bd3718c7479bc85b59d584108c056055024be16/.github/workflows/delivery-docker.yml#L7-L42?